brownie/data/default-config.yaml

```diff
@@ -12,6 +12,7 @@ networks:
         gas_price: 0
         reverting_tx_gas_limit: 6721975
         default_contract_owner: true
+        cmd_settings: {}
```

Comment:
Given this field is empty by default and you are checking for it's existence prior to working with it, is there a need to include it in the default config?

Reply:
The title in the docs says the default config shows all the available settings with their default values. I was on the fence about including it, but then the description in the docs would have to be changed slightly. Your call!

Reply:
Ahh very good point. Ok then..  can we make the setting `null` rather than `{}`? I think i just don't like the squiggly brackets.